lib/monitoring/logging-actor.js

```diff
@@ -0,0 +1,41 @@
+const {
```

Comment:
Strictly speaking, this filename is misleading, it isn't a "logging actor".

Reply:
You are completelly right here. The issue is that I started implementing the logging-actor to do this job.  But some weird circularity dependency problems was provoking that spawnStateless were still undefined wher I wanted to use it. Si I went for the simplest implementation for the moment until you give feedback. Some commented code you see is about this previous failed implementation. If you have some thouhgs about it I can update this with a Actor-based logging implementation.